### DISCUSSION TITLE
Blank page from gitpages

### DISCUSSION THREAD

**Discussion by Leandro-pixel** [community/community#110178] [Programming Help]
### Body

I'm getting blankpage after follow all the steps to publish my code, at first I was getting it cause I wasn't using hashrouter in my code(I'm using react-dom) but once I did the config and all the stuffs it works fine, but then I did the exactly same thing with another code and even if there's no console errors or error 404, I still getting the blank page and all that I saw that is different from the others repos is that this repo is saying that I'm 12 commits behind in my gitpage branch but in my terminal I can't pull or push anymore cause it's already all uo to date, I'm out of ideas 
[
![Screenshot_20240226-071342](https://github.com/community/community/assets/128332429/b3e07696-4fdf-4137-ae09-19cb1411b405)
]
The link for the repo: https://github.com/Leandro-pixel/Leandro_portfolio.git

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
The repo link I not found(404) look onto it

**Reply by Leandro-pixel** [reply]
You're right, I already changed the link and now is working, thanks!
